03/app.js

```diff
@@ -5,27 +5,57 @@ class Article extends React.Component {
     state = {
         comments: [],
```

Comment:
Trzeba tutaj dodać nowy element do `state`, aby przechowywał to co użytkownik wpisuje do pola formularza np. `content: ''`